src/MonthView/Weekdays.jsx

```diff
@@ -34,12 +35,13 @@ export default function Weekdays(props) {
       year, monthIndex, weekday - getDayOfWeek(beginOfMonth, calendarType),
     );
 
+    const weekdayClassName = `${className}__weekday`;
     const abbr = formatWeekday(locale, weekdayDate);
 
     weekdays.push(
       <div
         key={weekday}
-        className={`${className}__weekday`}
+        className={mergeClassNames(weekdayClassName, isWeekend(weekdayDate) && `${weekdayClassName}--weekend`)}
```

Comment:
`calendarType` is required in `isWeekend`.

```suggestion
        className={mergeClassNames(
          weekdayClassName, 
          isWeekend(weekdayDate, calendarType) && `${weekdayClassName}--weekend`
        )}
```